### PR TITLE
Add blog post announcing the upcoming OMERO.web Django upgrade work

### PR DESCRIPTION
**Pull request by sbesson**
Staged for review at https://snoopycrimecop.github.io/www.openmicroscopy.org/2022/02/17/omeroweb-django-upgrade.html

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

File: _posts/2022-02-17-omeroweb-django-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Since 5.6.0, OMERO.web has depended on Django 1.11.x.  This version left
+extended support in April 2020 [^1].  Django version conservatism has helped
+us establish a very fertile, stable environment for plugin developers.
+Furthermore, the usage of the plethora of Django functionality available in
+core OMERO.web is quite limited.
+
+Running a version of any software that the authors consider unsupported is
+risky; if any bugs of either class are found they will not be fixed upstream.
+However, we are not currently aware of any security or data loss bugs in th
```
Comment (will-moore) [inline _posts/2022-02-17-omeroweb-django-upgrade.md:19]: `th`

File: _posts/2022-02-17-omeroweb-django-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+latest available versions of Django 1.11.x that affect OMERO.web.
+
+For the future safety of the community as well as enhanced developer
+ergonomics we need to upgrade Django.  This was first attempted, targetting
+Django 2.2, during the summer of 2020 [^2].  Unfortunately, that work neve
```
Comment (will-moore) [inline _posts/2022-02-17-omeroweb-django-upgrade.md:24]: `neve`

File: _posts/2022-02-17-omeroweb-django-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+By the end of March 2022 we plan to have a release candidate version of
+OMERO.web with Django 3.2.x as a dependency to be made available for the
+community to evaluate.  We will notify the community as soon as this release
+candidate is available.  In anticipation of this we are recommending that
+__all__ plugin developers pin their `omero-web` dependencies to `<5.14.0` and
+make a release of their plugin as soon as possible.
+
+Plugin developers are encouraged to review the release notes, in particular
+the *Backwards incompatible* and *Features deprecated* for all major Django
+versions since 1.11.x to ensure they can be complaint with the new APIs.  They
```
Comment (will-moore) [inline _posts/2022-02-17-omeroweb-django-upgrade.md:40]: complaint  -> compliant

File: _posts/2022-02-17-omeroweb-django-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* [Django 2.1 release notes](https://docs.djangoproject.com/en/4.0/releases/2.1/)
+* [Django 2.2 release notes](https://docs.djangoproject.com/en/4.0/releases/2.2/)
+* [Django 3.0 release notes](https://docs.djangoproject.com/en/4.0/releases/3.0/)
+* [Django 3.1 release notes](https://docs.djangoproject.com/en/4.0/releases/3.1/)
+* [Django 3.2 release notes](https://docs.djangoproject.com/en/4.0/releases/3.2/)
+
+The OME team will be publishing a migration guide along with the release
+candidate to draw the attention of plugin developers to specific OMERO.web
+idioms and design patterns that __will__ change in this new major version.
+For those developers wishing to observe this process in real time may do so by
```
Comment (will-moore) [inline _posts/2022-02-17-omeroweb-django-upgrade.md:53]: Remove "For ". "Those..." reads better.

**Review by will-moore — COMMENTED** [review]
I wonder if we should include something about our plan to udgrade the omero-web plugins?
From initially looking at iviewer and figure, it seems that the number of changes required for the upgrade are quite minimal and we should probably release the plugins along with omero-web.

**Comment by sbesson** [comment]
Updated the date of the post and addressed all the typos. From yesterday's discussion, our intent is very much to be able to release as much as a full ecosystem (omero-web + omero-apps). This blog post was written before we had evidence about the feasibility of the apps upgrade so it omits a roadmap for the Web apps. 

Unless someone wants to propose an amendment, my inclination would be to get this in. Possible by the end of February, we will have a better idea of the impact and the release schedule and we can follow-up with another post about the apps and the release timeline?

**Review by jburel — APPROVED** [review]

**Pull request merged**